Dockerfile

```diff
@@ -27,7 +30,8 @@ EXPOSE 3000
 
 FROM base AS development
 
-CMD bundle exec rails server -b 0.0.0.0
+SHELL [ "/bin/bash", "-c" ]
+ENTRYPOINT bundle exec rails server -b 0.0.0.0
```

Comment:
This change (and the one at the end of the file) seems to prevent running an alternative command in the container, for example to run tests: https://github.com/ElixirTeSS/TeSS/actions/runs/15994949196/job/45116138794

was there a specific reason for it?